sprint-planning/readme.md

```diff
@@ -1,29 +1,83 @@
 # Planning
 
-## Content TODO's
+## Goals
 
-- Complete outline below
-- Determining team capacity
-- Tasking sprint stories
-  - size of tasks
-- WIP limits / dividing up into workstreams
+During the [sprint planning](https://www.agilealliance.org/glossary/sprint-planning), the team discusses and agrees on the scope for the upcoming sprint.
 
-## Goals
+**Goals are:**
+
+- Selecting the **stories** that will be implemented in the sprint.

```

Comment:
Suggestion: Keep the same form on all the list items throughout the document.  I would propose imperative style - i.e. 
- Select the **stories** that will be implemented in the sprint
- Estimate the **effort** required for the stories in the sprint
- Split the stories into **tasks**

Reply:
Changed in upcoming update